service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreatePostgresqlDbStep.java

```diff
@@ -53,6 +53,33 @@ protected void createResource(FlightContext context, ArmManagers armManagers) {
 
     var postgres = createServer(context, armManagers, postgresName);
 
+    // TODO can this be done with one call - or even better, in the same call as creating the
+    // server?
+    // Enable pg-bouncer
```

Comment:
Do we want to make `PGBOUNCER_ENABLED` a Landing Zone parameter with a default of `true`? Then gate setting these configs on that param being enabled (e.g. check `parametersResolver.getValue(…)`).

Just asking since I recall talking about making it parameterized, and potentially wanting to turn it off in the future to save $$.

Reply:
Great idea!  I'll see if I can do that

Reply:
This is an example of adding a new param I think: https://github.com/DataBiosphere/terra-landing-zone-service/pull/211

Reply:
Since all the params are strings... is there a preferred method to check if its "true" or is `"true".equalsIgnoreCase()` fine here?

Reply:
what about `Boolean.parseBoolean`?

Reply:
That's the sort of thing I wanted, thank you!

Reply:
... Which, apparently, is just `"true".equalsIgnoreCase()` under the covers
![image](https://github.com/DataBiosphere/terra-landing-zone-service/assets/5884792/7bb504be-25a7-443e-80cb-7bdd0cca4f11)